app/controllers/time_slot_controller.rb

```diff
@@ -41,10 +41,7 @@ def index
     @ubs = @appointment.try(:ubs)
 
     @time_slots = Ubs.all.where(active: true).each_with_object({}) do |ubs, memo|
-      memo[ubs] = ubs.available_time_slots(Time.zone.today...3.days.from_now, Time.zone.now)
-
-      # TODO: Refactor to not include these as available
-      memo.delete(ubs) if memo[ubs].empty?
+      memo[ubs] = ubs.available_time_slots(4)
```

Comment:
The method `available_time_slots` now has only a parameter: the number of available (business) days (including the actual day, if it's a available day).

Reply:
I think we shouldn't change the method signature. We should manipulate the `DateRange` before passing to it.
Also, the `4` here is a magic number. We could extract it to a constant like `SLOT_WINDOW_IN_DAYS`